content/markdown/01-basic-r/01-01-coding-basics.rmd

```diff
@@ -0,0 +1,118 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Coding Basics
+**Author:** Cole Brookson
+**Date:** 13 June 2022
+
+Fundamentally, our goal in using R is to ask our computer to help us answer the biological domain questions we have. In order for R to be useful to us in this sense, we have to know *how* to ask our computer to do these things. It is easiest to think of R as simply a interface between us and our computers. However, we must remember that *computers are not intelligent, they will only do EXACTLY what we tell them to do*. Therefore, we need to be specific about how we ask our computer to do things in service of answering our biological questions. This section covers some important concepts that will govern how we ask R to do things for us. 
+
+## R as a Calculator
+
+At the most basic level, we can use R as a (very complicated) calculator. We can perform all the most basic operations that we might be interested in. 
+
+The classic operations (addition, subtraction, multiplication, division) work much as you might expect 
+
+```{r}
+2 + 2
+3 - 2
+3 * 2 # multiplication is denoted by an asterisk
+6 / 2
+2 ^ 3 # exponentiation is denoted by this carrot
+```
+
+You can also ask R to help with more complicated operations such as:
+
+**Modulo operations** (getting the remainder of a division)
```

Comment:
Modulo might need some more explanation (or maybe I'm the only person who doesn't know what it is?)